data/campaigns/Sceptre_of_Fire/scenarios/2_Closing_the_Gates.cfg

```diff
@@ -157,6 +158,11 @@
             [/gold_carryover]
         [/objectives]
 
+        [music]
+            name=battle.ogg
+            append=yes
+        [/music]
+
```

Comment:
Think adding another track here could be nice.

Reply:
Sure!  Will add.